assets/scss/breadcrumb.scss

```diff
@@ -1,8 +1,15 @@
 .breadcrumb {
-  background-color: transparent;
+  background-color: $white;
   margin-top: -1rem;
-}
-
-.breadcrumb-item.active {
-  color: $mediumgrey;
+  border-top: 1px solid #e0e0e0;
+  border-bottom: 1px solid #e0e0e0;
```

Comment:
Would you mind verifying with the design team, if we indeed need to introduce a new grey? Perhaps we can stick to the ones we already have. But if so, it might be worth adding a variable?

Reply:
checked with design using $paper is fine